cli/tests/test_debug_toy_backend_init_command.rs

```diff
@@ -16,8 +16,10 @@ use crate::common::TestEnvironment;
 
 #[test]
 fn test_init_local_disallowed() {
+    // TODO(ilyagr): it is unclear whether `ui.allow-init-native` is useful now
+    // that `jj init` was renamed to `jj debug toy-backend-init`.
```

Comment:
I think we should just delete the config

Reply:
I think so too. I added a TODO because I'm not sure I want to do it in this PR.

Reply:
I'll resolve this, assuming that a TODO is OK for now.